### PR TITLE
Update CORS config to allow requests to /user/me from theguardian.com

### PR DESCRIPTION
**Pull request by davidrapson**
Update CORS config to allow requests to `/user/me` from theguardian.com. Thanks to @rtyley for helping me sort this out.

### CODE REVIEW

**Comment by jennysivapalan** [comment]
:+1: :mans_shoe:

File: frontend/conf/dev.conf [hunk 1]
```diff
@@ -46,4 +46,4 @@ touchpoint.backend.test=UAT
 staff.authorised.emails.groups = "membership.dev@guardian.co.uk"
 
 activity.tracking.bcrypt.salt="$2a$10$oL3umggRoHlQuvgoYpWDx."
```
Comment (feedmypixel) [inline frontend/conf/dev.conf]: should this salt be in here?

**Comment by feedmypixel** [comment]
Has the cors call to ```/user/me``` from profile ```/membership/edit``` been sanity checked?

**Comment by davidrapson** [comment]
Definitely, just sanity checked again now:

screen shot 2015-03-10 at 11 27 12

screen shot 2015-03-10 at 11 27 20

**Comment by davidrapson** [comment]
@jamesoram Mind giving this a once over to confirm that the membership tab on profile still displays for you too?

**Comment by jamesoram** [comment]
what is this fixing?

**Comment by davidrapson** [comment]
@jamesoram This is allowing us to request `/user/me` from the guardian.com. This involves changing the CORS headers, so we just need to double check that we don't also prevent `profile.theguardian.com` from requesting `/user/me`

**Comment by jamesoram** [comment]
it looks OK on my laptop but I don't think we will know 100%  until this is on prod.

**Comment by davidrapson** [comment]
@jamesoram Yeah, I'll be ready to rollback if needed but I'm confident in the change.

**Comment by jamesoram** [comment]
I'll be up in 10 mins

**Pull request merged**